content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -99,15 +118,20 @@ public/
 
 You need the database connection string to define the `DATABASE_URL` environment variable.
 
-If you are using Heroku, follow the steps below.
+If you are using Supabase, follow the steps below.
```

Comment:
I think this (and the steps) could be extracted to an external "Create a database on database providers" guides page. Then this is shorter here, an we can just continue with the connection string present.

(Maybe a future change)